python/katana/example_data.py

```diff
@@ -11,8 +11,15 @@
 __all__ = ["get_input", "get_input_as_url"]
 
 
-def get_inputs_directory(*, invalidate=False) -> Path:
+def get_inputs_directory(*, invalidate=False, rel_path=None) -> Path:
     inputs_dir = None
+    if rel_path == "csv-datasets":
+        paths_to_check = list(Path(__file__).parents) + list(Path.cwd().parents)
+        for path in paths_to_check:
+            csv_path = (path / "katana-enterprise" / "external").resolve()
+            if csv_path.exists():
+                return csv_path
```

Comment:
Why do you need this special case? I thought there was gonna be some way to register data sources like this so special cases are not needed.

Reply:
I moved the datasources needed for running tests against deployed katana to different repo. This is to fetch csv datasets when testing locally.